### DISCUSSION TITLE
0.7.0 for Zotero 7

### DISCUSSION THREAD

**Discussion by lifan0127** [lifan0127/ai-research-assistant#89] [Announcements]
This is an experimental pre-release of Aria for Zotero 7 (not compatible with Zotero 6). It offers the same functionalities as version 0.7.0 https://github.com/lifan0127/ai-research-assistant/releases/tag/0.7.0

<hr /><em>This discussion was created from the release <a href='https://github.com/lifan0127/ai-research-assistant/releases/tag/0.7.0-z7'>0.7.0 for Zotero 7</a>.</em>

**Comment by much-doge** [comment]
I have tested it briefly and it seems to be working flawlessly!

**Comment by dcgk** [comment]
Thank you so much for making this pre-release for Zotero 7 available. I am looking forward to testing it over the coming few days.
Upon installation and first opening it, may I ask you:
- In the preference pane and in the add-ons Manager it says: Version/Aria Build 1.0.0 (2024-04-14 20:04:54), should this not be 0.7.0 ? 
- for this experimental version, will updates through the Zotero add-ons Manager work, or should I come here and install them manually?
- Will you be adding a dark mode for the Aria interface to match the beautiful new Zotero 7 interface? That would be highly appreciated!

Thank you for providing this amazing plug-in for the Zotero community!

**Comment by bradclarkuk** [comment]
Hello, 

I'd love to use this for the upcoming new term at uni in september however i get the following error when trying to install this Zotero 7 compatible version:

The add-on "Aria" could not be installed. It may be incompatible with this version of Zotero.

Im on Zotero version: 7.0.0-beta.98+0b3eda5d8
Mac OS 15 

Any ideas?

**Reply by patrickheindel** [reply]
Same error here, thanks for the work on this project

Zotero 7.0.2
Mac OS 14.5
Macbook M1

**Comment by mkriem01** [comment]
thanks a lot for 0.7.0 for Zotero 7. Unfortunately the release (Apr 15) didn't work with me. Any idea of what could be the problem? best regards, Monika

**Comment by mkriem01** [comment]
Neither does the release from July 29

**Reply by lifan0127** [reply]
Hi @mkriem01 what is your Zotero version and is there any error message? Thanks!

**Reply by bradclarkuk** [reply]
@lifan0127 -    im running zotero 7.0.0 (-beta.98+0b3eda5d8) on mac and the error message i get when trying to install aria 0.7.2 is "The add-on "Aria" could not be installed. It may be incompatible with this version of Zotero."

**Reply by DeschutesMCLM** [reply]
@lifan0127 Any updates on ARIA for Zotero 7? I'm getting the same message (_The add-on "Aria" could not be installed. It may be incompatible with this version of Zotero._) when I try to install. Currently running Zotero 7.0.7.

P.S. This project is so cool—you are awesome for developing this!

**Reply by lifan0127** [reply]
Hi everyone Could you please try the following release? I tested it in Zotero 7.0.8 on Mac.

https://github.com/lifan0127/ai-research-assistant/releases/tag/v0.7.4

**Comment by tommasorosati** [comment]
Hi,
I always get this error: 

<pre>{
  "status": 404,
  "headers": {
    "cf-cache-status": "DYNAMIC",
    "cf-ray": "8c8597410a0e2305-ORD",
    "content-encoding": "br",
    "content-type": "application/json; charset=utf-8",
    "date": "Tue, 24 Sep 2024 20:40:39 GMT",
    "server": "cloudflare",
    "strict-transport-security": "max-age=31536000; includeSubDomains; preload",
    "vary": "Origin",
    "x-content-type-options": "nosniff",
    "x-firefox-spdy": "h2",
    "x-request-id": "req_e2ca51157fd72a662a0b973d6be1bf6e"
  },
  "error": {
    "message": "The model `gpt-4-0613` does not exist or you do not have access to it.",
    "type": "invalid_request_error",
    "param": null,
    "code": "model_not_found"
  },
  "code": "model_not_found",
  "param": null,
  "type": "invalid_request_error",
  "attemptNumber": 1,
  "retriesLeft": 6,
  "name": "Error",
  "message": "404 The model `gpt-4-0613` does not exist or you do not have access to it.",
  "stack": "Ae@jar:file:/<local-profile-dir>/extensions/aria@apex974.com.xpi!/chrome/content/scripts/aria.js:246:3146\nt@jar:file:/<local-profile-dir>/extensions/aria@apex974.com.xpi!/chrome/content/scripts/aria.js:246:3211\nIu@jar:file:/<local-profile-dir>/extensions/aria@apex974.com.xpi!/chrome/content/scripts/aria.js:246:4460\ngenerate@jar:file:/<local-profile-dir>/extensions/aria@apex974.com.xpi!/chrome/content/scripts/aria.js:246:3725\nmakeStatusError@jar:file:/<local-profile-dir>/extensions/aria@apex974.com.xpi!/chrome/content/scripts/aria.js:249:7509\nmakeRequest@jar:file:/<local-profile-dir>/extensions/aria@apex974.com.xpi!/chrome/content/scripts/aria.js:249:8402\n"
}</pre>

**Reply by lifan0127** [reply]
Hi @tommasorosati , have you tried other models such as gpt-4o?
